package/src/swagger_server/swagger/swagger.yaml

```diff
@@ -379,6 +379,8 @@ components:
               type: string
             message:
               type: string
+            password:
```

Comment:
Maybe use a different name to make it a bit more explicit that this is the password for the video call

Reply:
good idea, will change that